peer_mentoring/posts/models.py

```diff
@@ -16,7 +16,7 @@ class Post(models.Model):
     author = models.ForeignKey(User, on_delete=models.CASCADE)
     title = models.CharField(max_length=150)
     content = models.TextField(blank=False, null=True)
-    comments = models.ManyToManyField(Comment)
+    comments = models.OneToMany(Comment)
```

Comment:
sorry if I did not explain this well, for a one to many relation, I would use a models.ForeignKey field rather